bin/fastlane

```diff
@@ -4,6 +4,30 @@ if RUBY_VERSION < '2.0.0'
   abort("fastlane requires Ruby 2.0.0 or higher")
 end
 
+def self.windows?
+  # taken from: https://stackoverflow.com/a/171011/1945875
+  (/cygwin|mswin|mingw|bccwin|wince|emx/ =~ RUBY_PLATFORM) != nil
+end
+
+require 'bootsnap'
+require 'tmpdir'
+require 'fileutils'
+# env = ENV['RAILS_ENV'] || "development"
+require_relative "../fastlane/lib/fastlane/version.rb"
+cache_dir = "#{Dir.tmpdir}/fastlane/bootsnap-cache/" + Fastlane::VERSION
+unless File.directory?(cache_dir)
+  FileUtils.mkdir_p(cache_dir)
+end
+Bootsnap.setup(
+  cache_dir:            cache_dir, # Path to your cache
+  development_mode:     false, # env == 'development', # Current working environment, e.g. RACK_ENV, RAILS_ENV, etc
+  load_path_cache:      true, # Optimize the LOAD_PATH with a cache
+  autoload_paths_cache: false, # Optimize ActiveSupport autoloads with cache
```

Comment:
This comment about "ActiveSupport" doesn't seem relevant as we don't use `ActiveSupport`?

Reply:
Exactly - which is why this is set to `false`. 
Those comments are the default ones suggested by `bootsnap` documentation.

Reply:
I still think we should remove _this_ comment_ because we don't use the `ActiveSupport` gem so this doesn't make sense in our context even though the `bootsnap` documentation has it suggested